### PR TITLE
[runtime-audit-engine] improve docs

### LINKED ISSUE #4031: [runtime-audit-engine] Advanced Usage docs
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Use case. Why is this important?

Falco works tightly with the kernel, and sometimes it needs to be more obvious how to debug things.

### Proposed Solution

Provide documentation with internals and ways of debugging runtime audit engine agents behavior.

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by artbin**
## Description

Add more details to documentation and add advanced usage documentation.

## Why do we need it, and what problem does it solve?

We need to understand how to debug things in Falco.
Close #4031.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: runtime-audit-engine
type: feature
summary: Improve documentation and add advanced usage documentation.
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE_RU.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+Для получения метрик можно использовать PromQL-запрос `falco_events{}`:
+
+```shell
+kubectl -n d8-monitoring exec -it prometheus-main-0 prometheus -- \
+  curl -s http://127.0.0.1:9090/api/v1/query\?query\=falco_events | jq
+```
+
+## Эмуляция события Falco
+
+Есть два способа созданиясобытия Falco:
```
Comment (z9r5) [inline ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE_RU.md:70]:
```suggestion
Есть два способа создания события Falco:
```

File: ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE_RU.md
Comment (name212) [inline ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE_RU.md:1]: docs not sync with English version

File: ee/modules/650-runtime-audit-engine/openapi/config-values.yaml
Comment (name212) [inline ee/modules/650-runtime-audit-engine/openapi/config-values.yaml:1]: you should to add doc-ru-config-values.yaml with translate as [here](https://github.com/deckhouse/deckhouse/blob/main/modules/500-dashboard/openapi/doc-ru-config-values.yaml) for example

And values.yaml as here https://github.com/deckhouse/deckhouse/blob/main/modules/500-dashboard/openapi/values.yaml

File: ee/modules/650-runtime-audit-engine/openapi/config-values.yaml [hunk 2]
```diff
@@ -1,2 +1,6 @@
 type: object
-properties: {}
+properties:
+  debug:
```
Comment (name212) [inline ee/modules/650-runtime-audit-engine/openapi/config-values.yaml:3]: lets rename to `debugLogging`

File: ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+## Enabling debugging logs
+
+### Falco
+
+By default, the log level is set to `debug`.
+
+### Falcosidekick
+
+You can enable debugging logs by setting the `.Values.runtimeAuditEngine.debug` value to `true`.
```
Comment (name212) [inline ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE.md:13]: We need here to add exampe with module config and remove  `.Values.runtimeAuditEngine.debug`

File: ee/modules/650-runtime-audit-engine/docs/EXAMPLES_RU.md [hunk 4]
```diff
@@ -79,3 +79,10 @@ spec:
         (command=%proc.cmdline pid=%proc.pid connection=%fd.name sport=%fd.sport user=%user.name %container.info image=%container.image)
       priority: Notice
 ```
+
```
Comment (name212) [inline ee/modules/650-runtime-audit-engine/docs/EXAMPLES_RU.md:82]: add `Notifying when a shell is run in a container` example

File: ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE_RU.md
Comment (artbin) [reply ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE_RU.md:1]: done

File: ee/modules/650-runtime-audit-engine/openapi/config-values.yaml
Comment (artbin) [reply ee/modules/650-runtime-audit-engine/openapi/config-values.yaml:1]: done

File: ee/modules/650-runtime-audit-engine/openapi/config-values.yaml (hunk 2 above)
Comment (artbin) [reply ee/modules/650-runtime-audit-engine/openapi/config-values.yaml]: done

File: ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE.md (hunk 3 above)
Comment (artbin) [reply ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE.md]: done

File: ee/modules/650-runtime-audit-engine/docs/EXAMPLES_RU.md (hunk 4 above)
Comment (artbin) [reply ee/modules/650-runtime-audit-engine/docs/EXAMPLES_RU.md:82]: done

File: ee/modules/650-runtime-audit-engine/docs/EXAMPLES.md [hunk 5]
```diff
@@ -79,3 +79,29 @@ spec:
         (command=%proc.cmdline pid=%proc.pid connection=%fd.name sport=%fd.sport user=%user.name %container.info image=%container.image)
       priority: Notice
 ```
+
+## Notifying when a shell is run in a container
+
+```yaml
```
[the remark this answers is not in the document]
Comment (artbin) [reply ee/modules/650-runtime-audit-engine/docs/EXAMPLES.md:85]: done

**Review by z9r5 — APPROVED** [review]

**Review by z9r5 — APPROVED** [review]

File: ee/modules/650-runtime-audit-engine/docs/FAQ_RU.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Модуль runtime-audit-engine: примеры конфигурации"
+title: "Модуль runtime-audit-engine: FAQ                                                            "
```
Comment (name212) [inline ee/modules/650-runtime-audit-engine/docs/FAQ_RU.md:2]:
```suggestion
title: "Модуль runtime-audit-engine: FAQ"
```

File: ee/modules/650-runtime-audit-engine/docs/FAQ.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "The runtime-audit-engine module: usage"
+title: "The runtime-audit-engine module: FAQ         "
```
Comment (name212) [inline ee/modules/650-runtime-audit-engine/docs/FAQ.md:2]:
```suggestion
title: "The runtime-audit-engine module: FAQ"
```

File: ee/modules/650-runtime-audit-engine/openapi/config-values.yaml [hunk 8]
```diff
@@ -1,2 +1,6 @@
 type: object
-properties: {}
+properties:
+  debugLogging:
+    type: boolean
+    description: |
+      If true all outputs will print in stdout the payload they send.
```
Comment (name212) [inline ee/modules/650-runtime-audit-engine/openapi/config-values.yaml:6]: i think it does not match with russian doc

**Review by name212 — CHANGES_REQUESTED** [review]
some nits

File: ee/modules/650-runtime-audit-engine/openapi/config-values.yaml (hunk 8 above)
Comment (z9r5) [inline ee/modules/650-runtime-audit-engine/openapi/config-values.yaml:6]:
```suggestion
      If true, all events will be duplicated to stdout.
```

**Review by z9r5 — DISMISSED** [review]

File: ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+title: "The runtime-audit-engine module: advanced usage"
+---
+
+{% raw %}
+
+## Enabling debugging logs
+
+### Falco
+
+By default, the log level is set to `debug`.
```
Comment (yalosev) [inline ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE.md:11]: According to the template:
```
        {{- if .Values.runtimeAuditEngine.debugLogging }}
        - name: DEBUG
          value: "true"
        {{- end }}
```

and config
```
properties:
  debugLogging:
    type: boolean
    description: |
      If true all outputs will print in stdout the payload they send.
```

`Debug` is not a default. This sentence is invalid

**Review by yalosev — CHANGES_REQUESTED** [review]

File: ee/modules/650-runtime-audit-engine/openapi/config-values.yaml (hunk 8 above)
Comment (artbin) [reply ee/modules/650-runtime-audit-engine/openapi/config-values.yaml:6]: done

Comment (artbin) [reply ee/modules/650-runtime-audit-engine/openapi/config-values.yaml:6]: done

File: ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE.md (hunk 9 above)
Comment (artbin) [reply ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE.md]: I added `By default, the debug logging for `Falcosidekick` is disabled.` to the Falcosidekick section.

**Review by yalosev — DISMISSED** [review]

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Review by name212 — APPROVED** [review]

**Review by z9r5 — APPROVED** [review]

**Review by name212 — APPROVED** [review]

**Pull request merged**